test/verify/check-system-realms

```diff
@@ -609,7 +622,7 @@ ExecStart=/bin/true
         alice_cert = '\n'.join([line for line in alice_cert.splitlines() if not line.startswith("----")])
         # set up an IPA user with a TLS certificate; can't use "admin" due to https://pagure.io/freeipa/issue/6683
         ipa_machine.execute(r"""podman exec -i freeipa sh -exc '
-ipa user-add --first=Alice --last="Developer" alice
+ipa user-add --first=Alice --last="Developer" --shell=/bin/bash alice
```

Comment:
FTR, this is because the Terminal check below waits for an "alice" to appear in the prompt. With the default /bin/sh shell, the prompt is just `$`. I think it's more realistic to verify this with bash.